munch-match/src/App.test.js

```diff
@@ -1,49 +1,38 @@
-import { render, fireEvent} from '@testing-library/react';
+import { render, fireEvent } from '@testing-library/react';
 import React from 'react';
 import FoodOptionButtons from './components/foodOptionButtons';
 import Rating from "./components/rating"
 import Results from "./components/resultsLayout"
 import ResultCard from "./components/resultCard"
 import { getResults } from './services/MapsService.js';
+import foodChoices from './data/categories';
 
 // Testing if food buttons work as expected
 describe('FoodOptionButtons Component', () => {
-
-  it('calls onButtonClick with correct type for each button', () => {
-    const onButtonClickMock = jest.fn();
-    const { getByAltText } = render(<FoodOptionButtons onButtonClick={onButtonClickMock} />);
-    
-    const cafeButton = getByAltText('cafe');
-    fireEvent.click(cafeButton);
-    expect(onButtonClickMock).toHaveBeenCalledWith('cafe');
-
-    const burgerButton = getByAltText('burger');
-    fireEvent.click(burgerButton);
-    expect(onButtonClickMock).toHaveBeenCalledWith('burger');
-
-    const sushiButton = getByAltText('sushi');
-    fireEvent.click(sushiButton);
-    expect(onButtonClickMock).toHaveBeenCalledWith('sushi');
-
-    const fruitButton = getByAltText('fruit');
-    fireEvent.click(fruitButton);
-    expect(onButtonClickMock).toHaveBeenCalledWith('fruit');
-    
+  const onButtonClickMock = jest.fn();
+
+  foodChoices.forEach((choice, index) => {
+    it(`calls onButtonClick with correct type for ${choice}`, () => {
+      const { getByAltText } = render(<FoodOptionButtons onButtonClick={onButtonClickMock} />);
+      const choiceButton = getByAltText(choice);
+      fireEvent.click(choiceButton);
+      expect(onButtonClickMock).toHaveBeenCalledWith(foodChoices[index]);
```

Comment:
Can just have `toHaveBeenCalledWith(choice)` for clarity, I think. Then we don't need to include index in the `forEach` also.

Reply:
Nice catch on that one, I will update it